config.py

```diff
@@ -54,7 +54,10 @@ def __init__(self, fd, debug, arguments):
             self.__logger_file_path = config_parser.get(section, "logger_file_path")
             self.__persist_memorystate = config_parser.getboolean(section, "persist_memorystate")
         except (NoSectionError, NoOptionError) as e:
-            print str(e)
+            print(str(e))
```

Comment:
this is probably result of bad conflict resolution,
please remove second ``except`` branch, it is unreachable.